oxford.html

```diff
@@ -0,0 +1,250 @@
+                  programming basics. Or, learn to build your first server with Node.js.
+                  <br>
+                  <span class="application-text">
+                    <a href="https://nodegirlsoxford.typeform.com/to/VS1UxD">
+                      <span>Applications are open!</span>
+                    </a>
+                  </span>
+                </p>
+
+                <div class="row">
+                  <div class="col m12 sponsors-container">
+                    <a target="_blank" href="https://www2.oxfordshire.gov.uk/cms/content/oxfordshire-county-library-0">
+                      <img class="sponsor" target="_blank" src="https://assets.codebar.io/b//uploads/sponsor/avatar/409/library_logo.JPG" alt="Oxfordshire County Library">
+                    </a>
+                    <a target="_blank" href="https://www.deliveroo.co.uk">
+                      <img class="sponsor" target="_blank" src="images/sponsors/deliveroo.png" alt="Deliveroo">
+                    </a>
+                  </div>
+                </div>
+              </div>
+            </div>
+        </div>
+
+        <!--   Past Events Section   -->
+        <!-- <h3 class="header center green-text">Past events</h3>
+        <div class="past-events-container">
+
+        </div> -->
+        <div class="contact-chapter-container">
+          <h4 class="header center green-text">Contact Oxford</h4>
+          <ul class="social-icons vertical">
+            <li class="social-icons__icon">
+              <a target="_blank" class="black-text" href="https://twitter.com/nodegirlsoxford">
+                <i class="fa fa-twitter fa-3x"></i>
+                <h5 class="header col s12 light">
+                  @nodegirlsoxford
+                </h5>
+              </a>
+            </li>
+            <li class="social-icons__icon">
+              <a class="black-text" href="mailto:oxford@nodegirls.com">
+                <i class="fa fa-envelope-o fa-3x"></i>
+                <h5 class="header col s12 light">
+                  oxford@nodegirls.com
+                </h5>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+    </div>
+
+  </main>
+
+
+
+  <footer class="page-footer green" id="footer">
+    <div class="container">
+      <div class="row">
+        <div class="col l6 s12 footer__contact">
+          <h5 class="white-text">Contact Us</h5>
+          <p class="white-text">If you have any questions, would like to become a mentor, sponsor an event or want to be involved in some way,
+            drop us a line to
+            <a href="mailto:hello@nodegirls.com" class="footer__mailto">hello@nodegirls.com</a>.
+          </p>
+        </div>
+
+        <div class="col l6 s12">
+          <ul class="social-icons">
+            <li class="social-icons__icon">
+              <a target="_blank" class="white-text" href="https://twitter.com/nodegirlslondon">
```

Comment:
We should make a note to update this across the site to just @nodegirls as opposed to the London account